nameko/cli/code.py

```diff
@@ -0,0 +1,38 @@
+from __future__ import absolute_import
+
+from code import InteractiveConsole
+
+
+class RaisingInteractiveConsole(InteractiveConsole):  # pragma: no cover
```

Comment:
You might be able to `no cover` the whole module in one go.

I think it'd also be worth putting a little note explaining why it's excluded from coverage.

Reply:
Is there a way to add `no cover` to the whole module? I think excluding the whole file can only be done by specifying the `omit` parameter for coverage but in that case it will not be visible in the module that it is omitted from coverage. I agree that it should be explained why `no cover` is there but perhaps it would be better to be explicit and keep the current `no cover`s with some comment next to them.